modules/java_api/build.gradle

```diff
@@ -51,7 +51,7 @@ sourceSets {
     test {
         java {
             srcDirs = ["."]
-            include "tests/*.java"
+            include "tests/api_2_0/*.java", "tests/compatibility/*.java"
```

Comment:
Please create a folder only for `compatibility` - 2.0 can be inside the `test` folder